pom.xml

```diff
@@ -284,7 +283,7 @@
                                         <version>1.8.0</version>
                                     </requireJavaVersion>
                                     <requireMavenVersion>
-                                        <version>3.1.0</version>
+                                        <version>3.3.9</version>
```

Comment:
Not necessary, but if anyone wants to build with javac of Java 9 they'll need to upgrade.
I am looking at it from adding a regression build with Java 9.